src/printer.js

```diff
@@ -3331,6 +3333,13 @@ function printArrayItems(path, options, printPath, print) {
   return concat(printedElements);
 }
 
+function getJSXRawValue(node) {
```

Comment:
This function name is not correct since you are also converting it to be a double quote. I would just inline those two lines above so you don't have to find a name for it.